ReleaseNotes.md

```diff
@@ -1,3 +1,44 @@
+### New in 0.21.0 (released 2016/07/29)
+
+**Features**
+
+This release adds support across Octokit.net for providing the repository Id
+rather than a name/owner pair. The repository Id does not change when transferring
+ownership of a repository, and is more robust for API callers. This work
+was lead by @dampir as part of Google Summer of Code 2016.
+
+ - Added new fields for Deployment and DeploymentStatus preview API - #1365 via @ErikSchierboom
+ - Added new fields for signature verification to Git Data Tag API - #1420 via @Sarmad93
+ - Added new fields for GitHub Pages preview API - #1421 via @dampir
+
+**Fixes**
+
+ - Fix serialization of enum value attributes - #1402 via @maddin2016
+ - Fix searching for repositories with underscore in name - #1418 via @dsplaisted, @shiftkey
+
+**Other**
+
+ - Clarified obsolete warnings for Protected Branch preview API - #1428 via @ryangribble
+ - Remove Obsolete items - #1422 via @ryangribble
+
+**Breaking Changes**
+
+After a long grace period, #1422 has removed these obsoleted members. These features
+exist in other parts of the API surface:
+
+ - `I(Observable)GitHubClient.Release`
+ - `I(Observable)GitHubClient.Notification`
+ - `I(Observable)GitHubClient.GitDatabase`
+ - `I(Observable)GitHubClient.SshKey`
+ - `I(Observable)GitHubClient.Repository.RepositoryComments`
+ - `I(Observable)GitHubClient.Repository.CommitStatus`
+ - `I(Observable)GitHubClient.Repository.RepoCollaborators`
+ - `I(Observable)GitHubClient.Repository.Commits`
+
+This method has also been removed, but is no longer supported through the API: 
```

Comment:
The `, but` here feels like the second clause contrasts with the first, but it really goes hand in hand with it. How about,

"This method is no longer supported through the API and has been removed from Octokit.net." or something like that?